packages/commonwealth/client/scripts/views/pages/Communities/Communities.tsx

```diff
@@ -231,187 +260,265 @@ const CommunitiesPage = () => {
+                />
+              )}
+              {filters.withCommunityType && (
+                <CWTag
+                  label={filters.withCommunityType}
+                  type="filter"
+                  onCloseClick={removeCommunityTypeFilter}
+                />
+              )}
+              {filters.withNetwork && (
+                <CWTag
+                  label={filters.withNetwork}
+                  type="filter"
+                  onCloseClick={removeChainNetworkFilter}
+                />
+              )}
+              {filters.withCommunityEcosystem && (
+                <CWTag
+                  label={filters.withCommunityEcosystem}
+                  type="filter"
+                  onCloseClick={removeCommunityEcosystemFilter}
+                />
+              )}
+              {filters.withEcosystemChainId && (
+                <CWTag
+                  label={
+                    Object.entries(communityChains).find(
+                      ([_, v]) => filters.withEcosystemChainId === v,
+                    )?.[0] as string
+                  }
+                  type="filter"
+                  onCloseClick={removeEcosystemChainIdFilter}
+                />
+              )}
+              {filters.withStakeEnabled && (
+                <CWTag
+                  label="Stake"
+                  type="filter"
+                  onCloseClick={removeStakeFilter}
+                />
+              )}
+              {filters.withTagsIds &&
+                filters.withTagsIds.map((id) => (
+                  <CWTag
+                    key={id}
+                    type="filter"
+                    label={(tags || []).find((t) => t.id === id)?.name || ''}
+                    onCloseClick={() => removeTagFilter(id)}
+                  />
+                ))}
+              <FiltersDrawer
+                isOpen={isFilterDrawerOpen}
+                onClose={() => setIsFilterDrawerOpen(false)}
+                filters={filters}
+                onFiltersChange={(newFilters) => setFilters(newFilters)}
               />
-            )}
-            {filters.withStakeEnabled && (
-              <CWTag
-                label="Stake"
-                type="filter"
-                onCloseClick={removeStakeFilter}
+            </div>
+            <CommunitiesTabContent
+              isLoading={isLoading}
+              isInitialCommunitiesLoading={isInitialCommunitiesLoading}
+              communitiesList={communitiesList}
+              containerRef={containerRef}
+              filters={filters}
+              historicalPrices={historicalPrices}
+              ethUsdRate={Number(ethUsdRate)}
+              setSelectedCommunityId={setSelectedCommunityId}
+              hasNextPage={hasNextPage}
+              fetchMoreCommunities={fetchMoreCommunities}
+            />
+          </>
+        )}
+
+        {/* Default fallback if tab is not recognized */}
+        {!TAB_VIEWS.find((tab) => tab.value === activeTab) && (
```

Comment:
this is redundant and repeats a lot of code. We should just with useEffect if the tab value is correct and if not, redirect user to 'all'.